public/app/features/explore/TraceView/components/TracePageHeader/TracePageHeader.tsx

```diff
@@ -176,20 +196,78 @@ export default function TracePageHeader(props: TracePageHeaderEmbedProps) {
     return { ...rest, value: renderer(trace, timeZone, styles) };
   });
 
+  const spanWithHeaderTags = getSpanWithHeaderTags(trace.spans);
+
   const title = (
     <h1 className={styles.TracePageHeaderTitle}>
       <TraceName traceName={getTraceName(trace.spans)} />{' '}
       <small className={cx(styles.TracePageHeaderTraceId, uTxMuted)}>{trace.traceID}</small>
     </h1>
   );
 
+  const newTitle = (
+    <h1 className={cx(styles.TracePageHeaderTitle)}>
+      <TraceName traceName={getTraceName(trace.spans)} />
+      <small>
+        <span className={styles.divider}>|</span>
+        {formatDuration(trace.duration)}
+      </small>
+    </h1>
+  );
+
+  const method = spanWithHeaderTags?.tags.filter((tag) => {
+    return tag.key === 'http.method';
```

Comment:
Maybe we could move this extraction into the `getSpanWithHeaderTags` function? That way we can keep the string constants there and we don't have to do the same filtering of the `tags` list twice.